Indicators/IndicatorExtensions.cs

```diff
@@ -61,6 +61,14 @@ public static bool Update(this IndicatorBase<IndicatorDataPoint> indicator, Date
             return second;
         }
 
+        public static SimpleMovingAverage SMA<T, TSecond>(this IndicatorBase<T> left, int period, bool waitForFirstToReady = true)
+            where T :BaseData
+            where TSecond : IndicatorBase<IndicatorDataPoint>
+        {
+            SimpleMovingAverage _smaADX = new SimpleMovingAverage("SMA of " + left.Name, period).Of(left);
+            return _smaADX;
```

Comment:
Typically _ is reserved for class privates, "smaADX" is fine here. The underscore in the issue description was my mistake :)